<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scatterDialog</class>
 <widget class="QDialog" name="scatterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cmbopc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_X">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>801</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_Y">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>590</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>86</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
